BridgeSDK/BridgeAPI/BridgeAPITypes/MachineGeneratedDoNotEdit/_SBBUserSessionInfo.m

```diff
@@ -157,6 +157,11 @@ - (void)updateWithDictionaryRepresentation:(NSDictionary *)dictionary objectMana
 
     self.signedMostRecentConsent = [dictionary objectForKey:@"signedMostRecentConsent"];
 
+    // If we're just creating a stub object to be populated later with actual JSON, don't try to create
+    // any included subobjects at this point. We can tell because the dictionary will consist of only the
+    // "type" key and nothing else.
+    BOOL creatingObjectBeforePopulating = (dictionary.count == 1) && dictionary[@"type"];
```

Comment:
ugh, OK. It's only a C idiom I've been using since 1985... you kids these days and your fancy "readable" code. ;)